packages/paste-website/src/components/Footer.tsx

```diff
@@ -0,0 +1,146 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import {themeGet} from 'styled-system';
+import {Link} from 'gatsby';
+
+import {Anchor} from '@twilio-paste/anchor';
+import {TWILIO_BLUE} from '../constants';
+import TwilioIcon from './svg/twilio-icon';
+
+const StyledFooter = styled.footer`
+  margin-top: 96px;
+  padding: 120px ${themeGet('space.space200')} 120px;
+  background-color: #f9fafb;
```

Comment:
This could be `$color-background-row-striped` (`backgrounds.colorBackgroundRowStriped`) or potentially `$color-gray-10` (`colors.colorGray10`) tokens. I'm not sure how I feel about the grey-scale tokens and was potentially going to remove them. 

I kind of wanted very specific tokens for specific purposes, but this might (re)convince me that the any purpose gray tokens are _"OK_"?